src/Uno.UI.RemoteControl.Server/Uno.UI.RemoteControl.Server.csproj

```diff
@@ -25,7 +32,7 @@
 
 			<_baseNugetPath Condition="'$(USERPROFILE)'!=''">$(USERPROFILE)</_baseNugetPath>
 			<_baseNugetPath Condition="'$(HOME)'!=''">$(HOME)</_baseNugetPath>
-			<_TargetNugetFolder>$(_baseNugetPath)\.nuget\packages\$(_OverridePackageId)\$(UnoNugetOverrideVersion)\tools\rc\host\$(TargetFramework)</_TargetNugetFolder>
+			<_TargetNugetFolder>$(_baseNugetPath)\.nuget\packages\$(_OverridePackageId).RemoteControl\$(UnoNugetOverrideVersion)\tools\rc\host\$(TargetFramework)</_TargetNugetFolder>
```

Comment:
Instead of adding `.RemoteControl` here, consider updating `_OverridePackageId` value itself.

Reply:
I copied this from other projects where it was done that way. Across the whole solution, there's a lot of inconsistency in how overridden package Ids are defined. I didn't want to get distracted with trying to standardize these as part of this PR though.